Modules/mathmodule.c

```diff
@@ -3295,6 +3295,25 @@ math_comb_impl(PyObject *module, PyObject *n, PyObject *k)
 }
 
 
+/*[clinic input]
+math.nextafter
+
+    x: float
+    y: float
```

Comment:
Oh. I didn't notice, that's a C float, whereas I mean a "Python float". In fact, I should use "double" here.

I will fix it once the buildbot tests will complete.